config/locales/en.yml

```diff
@@ -288,6 +288,7 @@ en:
       body_html: |
         Click the button below to log in to your Brave Rewards account. This email contains a private access link that can only be used once.
       button: Log in to Brave Rewards
+      explicit_url: If the above button doesn't work, paste this link in your browser %{url}
```

Comment:
Can we make the link clickable?

Reply:
The feature request was to not allow a clickable link. It's to allow someone to copy and paste from their preferred browser (in particular on iOS)

Reply:
On my iPhone when I can't open a link in my email client I press and hold the link and click "Copy URL" rather than just click it. 

When I have to manipulate the little pipes to click the link it's annoying but doable